src/pages/home/HomePage.tsx

```
@@ -13,6 +13,8 @@ import Divider from '@/shared/ui/Divider';
 import Flex from '@/shared/ui/Flex';
 import HomeExpenseItem from './ui/HomeExpenseItem';
 import * as S from './HomePage.style';
+import Button from '@/shared/ui/Button';
```

ESLint

[error] `@/shared/ui/Button` import should occur before import of `./ui/HomeExpenseItem`

(import/order)